docker/build-tools/Dockerfile

```diff
@@ -76,16 +76,21 @@ FROM ubuntu:xenial as base_os_context
 
 # hadolint ignore=DL3008
 RUN apt-get update && apt-get install -y --no-install-recommends \
-    make \
     ca-certificates \
+    curl \
+	g++ \
+	gcc \
     git \
+	libc6-dev \
+	make \
+	pkg-config \
```

Comment:
all these deps are not needed.  gcc and libc6-dev do it for me.  take a look at my PR on this (I also added a small bit of comments)